services/ymax-planner/src/engine.ts

```diff
@@ -53,6 +53,12 @@ import { makeWorkPool } from '@agoric/internal/src/work-pool.js';
 
 import type { CosmosRestClient } from './cosmos-rest-client.ts';
 import type { CosmosRPCClient, SubscriptionResponse } from './cosmos-rpc.ts';
+import { type KeyValueStore } from './kv-store.ts';
+import {
+  handlePendingTx,
+  type EvmContext,
+  type HandlePendingTxOpts,
+} from './pending-tx-manager.ts';
```

Comment:
I'd like to avoid `import { …, type $name, … }` patterns, and have been cleaning them up as the chance presents itself.
```suggestion
import type { KeyValueStore } from './kv-store.ts';
import { handlePendingTx } from './pending-tx-manager.ts';
import type { EvmContext, HandlePendingTxOpts } from './pending-tx-manager.ts';
```